roles/azimuth_caas_operator/defaults/main.yml

```diff
@@ -119,23 +119,7 @@ azimuth_caas_stackhpc_slurm_appliance_metadata_url: >-
 # The ID of the image to use with the StackHPC Slurm appliance
 # Support the old name for backwards compatibility
 # By default, use the openhpc image from community images if available
-azimuth_caas_stackhpc_slurm_appliance_image: >-
-  {{-
-    azimuth_caas_stackhpc_slurm_appliance_rocky8_image
-    if azimuth_caas_stackhpc_slurm_appliance_rocky8_image is defined
-    else (
-      community_images_image_ids.openhpc
-      if (
-        community_images_image_ids is defined and
-        'openhpc' in community_images_image_ids
-      )
-      else (
-        undef(hint = 'azimuth_caas_stackhpc_slurm_appliance_image is required')
-        if azimuth_caas_stackhpc_slurm_appliance_enabled
-        else None
-      )
-    )
-  }}
+azimuth_caas_stackhpc_slurm_appliance_image: 7ca99016-c342-4557-8a8d-9a856e934b58
```

Comment:
Lets just update to the latest published slurm image please:
https://github.com/azimuth-cloud/ansible-collection-azimuth-ops/commit/a8837e9c7c5d2d53c6aa1db4224576d6b15fd79b#diff-b519ef05a7257930be00bc4a48df8faf25dac42a1f3446178b885e27d262776eL82